### ISSUE TITLE
relative measurement

### ISSUE THREAD

**Issue by Eddylib** [aau-cns/mars_lib#13]
Is mars_lib lost support for relative measurement compared with ethzasl_msf? I haven't seen any part of relative measurement in source  code.

**Comment by Chris-Bee** [comment]
Hello @Eddylib thanks for reaching out. Could you please explain what you mean by relative measurement in your case?

**Comment by Eddylib** [comment]
I want to fuse vio (vision imu odometry) pose estimation with IMU measurement. I noticed in mars_lib it can be done with a absolute state measurement with sensor states Tvw and scale which transform vio estimate to fused coordinate frame. The vio is a vision odometry and the Tvw and scale might change by time, but the determinant of covariance of Tvw will become smaller and smaller and  it cause Tvw becoming increasingly difficult to update.

So In my case, I want to use vio as a relative pose estimate between time point of camera pose and fuse it to core state. The method was mentioned in msf paper "A Robust and Modular Multi-Sensor Fusion Approach Applied to MAV Navigation". But there is no relative part in mars_lib.

**Comment by Chris-Bee** [comment]
Absolutely, we do use the vision sensor update for the integration of most VIO solutions. However, if you are planning to only use vision and IMU, then the vision reference will be unobservable, which is expected. I will have to check the reference to MSF with respect to the relative measurement. Just so we are on the same page, are you refering to Section 1-C?
